heapster-saw/src/Verifier/SAW/Heapster/Parser.y

```diff
@@ -205,7 +205,9 @@ permOffset ::                                   { Maybe AstExpr         }
 
 funPerm ::                                      { AstFunPerm }
   : '(' ctx ')' '.' funPermList '-o' funPermList
-                                                { AstFunPerm (pos $6) $2 $5 $7 }
+                                                { AstFunPerm (pos $6) $2 $5 [] $7 }
+  | '(' ctx ')' '.' funPermList '-o' '(' ctx ')' funPermList
```

Comment:
```suggestion
  | '(' ctx ')' '.' funPermList '-o' '(' ctx ')' '.' funPermList
```
Missing a dot? There should probably be an example added that tests this new syntax as well...